src/components/chart/_macro.njk

```diff
@@ -125,6 +126,24 @@
                 })
             }}
         {% endif %}
+
+        {% if params.footnotes %}
+            {% set footnotesId = ["footnotes--", params.id] | join %}
+
+            {{
+                onsDetails({
+                    "id": footnotesId,
+                    "title": params.footnotes.title,
+                    "content": params.footnotes.content,
+                    "headingLevel": headingLevel + 2,
+                    "classes": "ons-u-mt-xl",
+                    "attributes": {
+                        "data-footnotes": footnotesId
+                    }
```

Comment:
Just wondering whats this attribute used for? If its just for the css selector in the tests we could use something else to select the element

Reply:
Yep it's used for the tests. In general I always avoid using class names to target DOM elements, as they could change when styling is updated and unexpectedly impact the JS behaviour. But let me know if you use a different convention in the DS.

Reply:
Could you not use the id?

Reply:
I've updated that now.